onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -779,6 +779,10 @@ def main():
     enable_onnxruntime = "onnxruntime" in args.engines
     enable_tensorflow = "tensorflow" in args.engines
 
+    if enable_torch2 and torch.__version__ < "2.0.0":
```

Comment:
use packaging.version to compare version like `version.parse(torch.__version__) < version.parse("2.0.0")`

Reply:
Fixed